scripts/clear-all-fault-leds.sh

```diff
@@ -68,7 +68,7 @@ then
         GetSubTreePaths sias "/xyz/openbmc_project/inventory" 0 1 "xyz.openbmc_project.State.Decorator.OperationalStatus" \
         | sed  's/ /\n/g' | tail -n+3 | awk -F "\"" '{print $2}' | while read -r line
     do
-        echo "$line" | grep "unit\|core" >/dev/null
+        echo "$line" | grep "powersupply\|dimm\|dcm\|fan" >/dev/null
```

Comment:
With fans and power supplies in this list, the customer will need to physically replace them before the functional status will ever get set back true,  so the health status in the GUI will remain critical until then.  Maybe they won't notice/care.

You added those because of this defect? https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=460286?

I guess you're OK with this Jinu then I am.  We may need to revisit it if a field defect pops up.